lib/Api/Companies.php

```diff
@@ -31,6 +31,14 @@ class Companies extends Api
     protected $itemName = 'company';
 
     /**
+     * @var array
+     */
+    protected $bcRegexEndpoints = array(
+        'companies/(.*?)/contact/(.*?)/add'    => 'companies/$1/contact/add/$2', // 2.6.0
+        'companies/(.*?)/contact/(.*?)/remove' => 'companies/$1/contact/remove$2' // 2.6.0
```

Comment:
Good catch. This revealed that tests were not catching the BC urls passed the first key. Got that fixed.